tensorflow/tools/docker/Dockerfile.devel

```diff
@@ -92,7 +92,7 @@ WORKDIR /tensorflow
 ENV CI_BUILD_PYTHON python
 
 RUN tensorflow/tools/ci_build/builds/configured CPU \
-    bazel build -c opt tensorflow/tools/pip_package:build_pip_package && \
+    bazel build -c opt --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" tensorflow/tools/pip_package:build_pip_package && \
```

Comment:
Thanks for sending the patch, @aleist. Just a style nitpick: Can you break and indent lines as done consistently in the files? Thanks.